README.md

```diff
@@ -1,12 +1,11 @@
 # Business Site
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The assignment was to create a responsive business site with an image or a video as a header. There should also be a form and a accordion which included some javaScript.
 
 ## The problem
 
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
```

Comment:
This one is still the template - would be interesting to know more like on the demo : D